lib/ex_phone_number/mix/tasks/update_metadata.ex

```diff
@@ -8,14 +10,31 @@ defmodule Mix.Tasks.UpdateMetadata do
   @resources_directory "resources"
 
   defmodule GitHub do
-    use Tesla
+    def latest_release(repo) do
+      ensure_tesla_loaded!()
```

Comment:
I think we could simply not define the task if Tesla is missing - it's an internal task anyway:

```elixir
if Code.ensure_loaded?(Tesla) do
  defmodule MixTasks.UpdateMetadata do
    # ...
  end
end
```

Reply:
Yeah, that makes sense... I've updated the code. What do you think?